src/software/ai/hl/stp/tactic/test_tactics/move_test_tactic.h

```diff
@@ -18,6 +18,13 @@ class MoveTestTactic : public Tactic
      */
     explicit MoveTestTactic(bool loop_forever = false);
 
+    /**
+     * Updates the world parameters for this ChipTactic.
+     *
+     * @param World The current state of the world
+     */
+    void updateWorldParams(const World& world) override;
```

Comment:
it's not necessary to have a function comment since we're overriding

Reply:
resolved